examples/k8s/hz-go-service/go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/nevzatseferoglu/hz-go-service
+
+go 1.17
+
+require (
+	github.com/gorilla/mux v1.8.0
+	github.com/hazelcast/hazelcast-go-client v1.1.2-0.20220131082557-ec272b3372a8
```

Comment:
We may prefer to use a static version such as `v1.2.0`

Reply:
Switched.

Reply:
Updated with `1.3.0`, [9c4baec](https://github.com/hazelcast/hazelcast-go-client/pull/743/commits/9c4baecc96452e673ce3249504f10ce2efa3d810).